google-cloud-spanner/pom.xml

```diff
@@ -104,21 +114,19 @@
           </execution>
         </executions>
       </plugin>
-
-  <!--Configurations for native image tests-->
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-surefire-plugin</artifactId>
-        <configuration>
-          <systemPropertyVariables>
-            <spanner.testenv.config.class>${spanner.testenv.config.class}</spanner.testenv.config.class>
-            <spanner.testenv.instance>${spanner.testenv.instance}</spanner.testenv.instance>
-            <spanner.gce.config.project_id>${spanner.gce.config.project_id}</spanner.gce.config.project_id>
-            <spanner.testenv.kms_key.name>${spanner.testenv.kms_key.name}</spanner.testenv.kms_key.name>
-          </systemPropertyVariables>
-          <forkedProcessTimeoutInSeconds>3000</forkedProcessTimeoutInSeconds>
-        </configuration>
-      </plugin>
+<!--      <plugin>-->
+<!--        <groupId>org.apache.maven.plugins</groupId>-->
```

Comment:
Should we just remove these commented config? Or do you intend to revise the PR later @mpeddada1 ?

Reply:
Thanks @ansh0l! I kept these in for testing purposes but yes, we remove these before merging the PR.

Reply:
All set. I've removed this comment from the pom.xml.